Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift

```diff
@@ -123,6 +123,8 @@ final class IssueCommentHtmlCell: UICollectionViewCell, ListBindable, UIWebViewD
     }
 
     func webViewDidFinishLoad(_ webView: UIWebView) {
+        webView.stringByEvaluatingJavaScript(from: "document.documentElement.style.webkitUserSelect='none'")
+        webView.stringByEvaluatingJavaScript(from: "document.documentElement.style.webkitTouchCallout='none'")
```

Comment:
Shouldn't this be added into the HTML template?

https://github.com/rnystrom/GitHawk/blob/master/Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift#L25-L53